src/main/java/io/quarkus/gizmo2/creator/BlockCreator.java

```diff
@@ -2866,8 +2875,9 @@ default Expr mapOf(Expr... items) {
      * @param throwable the throwable (must not be {@code null})
      * @param suppressed the suppressed throwable (must not be {@code null})
      */
-    // TODO: withThrowable(throwable).addSuppressed(suppressed) ?
-    void addSuppressed(Expr throwable, Expr suppressed);
+    default void addSuppressed(Expr throwable, Expr suppressed) {
+        withThrowable(throwable).addSuppressed(suppressed);
+    }
```

Comment:
I would actually remove this method, I think?

Reply:
Either way is OK by me.

Reply:
Removed.